.gitignore

```diff
@@ -2,3 +2,4 @@
 .jekyll-cache/
 .sass-cache/
 public/
+.DS_Store
```

Comment:
This should be in your global `~/.gitignore`. :grimacing: 

See https://github.com/github/gitignore/blob/master/Global/macOS.gitignore for an example (assuming you're using macOS, of course).

Reply:
While I agree (and I'll do that), I think adding it to the project gitignore doesn't hurt...?